packages/react-storage/src/components/StorageBrowser/controls/ActionStartControl.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { ControlProps } from './types';
+import { useResolvedComposable } from './hooks/useResolvedComposable';
+import { ActionStart } from '../composables/ActionStart';
+import { useActionStart } from './hooks/useActionStart';
+import { ViewElement } from '../context/elements';
+
+export const ActionStartControl = ({
+  className,
+}: ControlProps): React.JSX.Element | null => {
+  const { props } = useActionStart();
+  const ResolvedActionStart = useResolvedComposable(ActionStart, 'ActionStart');
+  if (!props) {
```

Comment:
Once we remove props from [here](https://github.com/aws-amplify/amplify-ui/pull/5935/files#r1815405481) I think we'll need to do the same here. (also see [Reference comment](https://github.com/aws-amplify/amplify-ui/pull/5939/files#r1811705428))